### PR TITLE
Update build result colors

### PR DESCRIPTION
**Pull request by bgeuken**
Before:

screenshot from 2018-11-28 10-11-01

After:

screenshot from 2018-11-28 10-15-32

### CODE REVIEW

File: src/api/app/assets/stylesheets/webui2/build-results.scss [hunk 1]
```diff
@@ -17,3 +17,31 @@
...
+.build-state-signing, .build-state-finished {
+  color: $dark;
+}
+
+.build-state-unresolvable, .build-state-broken {
+  color: $danger;
+}
+
+.build-state-disabled, .build-state-block {
+  @extend .text-black-50;
```
Comment (eduardoj) [inline src/api/app/assets/stylesheets/webui2/build-results.scss]: Isn't it blocked instead block?

Comment (Ana06) [reply src/api/app/assets/stylesheets/webui2/build-results.scss]: right!

Comment (bgeuken) [reply src/api/app/assets/stylesheets/webui2/build-results.scss]: Good catch

**Comment by bgeuken** [comment]
The package builds were finished, but the state not updated. Hence I merged it now

**Comment by hennevogel** [comment]
Package builds are not required...

**Pull request merged**